index.js

```diff
@@ -72,4 +80,4 @@ function readCsv(filename, headers) {
       reject(err);
     }
   });
-}
+}
```

Comment:
Add empty newline.

Reply:
@davis-trem 

This may be obvious but just wanted to double confirm. All I am doing here for the readCsv function is creating a new line in between the closing bracket for the readCsv function and the returned promise? 

Basically, creating some space to identify the readCsv function closing bracket?

Reply:
Nah, add a newline at the very end of the file. I'm not sure why specifically we do it but it's something my team has always marked to do. Also github shows a red no-no if your file doesn't end with an empty line. 
@getromandev